srcpkgs/pipewire/template

```diff
@@ -1,24 +1,26 @@
 # Template file for 'pipewire'
 pkgname=pipewire
-version=0.3.28
+version=0.3.31
 revision=1
 build_style=meson
 configure_args="-Dman=enabled -Dgstreamer=enabled -Ddocs=enabled -Dsystemd=disabled
  -Dbluez5=enabled -Dffmpeg=enabled -Dpipewire-alsa=enabled -Dpipewire-jack=enabled
- -Dvulkan=enabled -Dudevrulesdir=/usr/lib/udev/rules.d"
+ -Dvulkan=enabled -Dudevrulesdir=/usr/lib/udev/rules.d -Db_ndebug=false"
 hostmakedepends="doxygen graphviz pkg-config xmltoman gettext"
 makedepends="ffmpeg-devel gst-plugins-base1-devel jack-devel sbc-devel v4l-utils-devel
- libva-devel libbluetooth-devel ncurses-devel libopenaptx-devel fdk-aac-devel
+ libva-devel libbluetooth-devel ncurses-devel libopenaptx-devel libusb-devel fdk-aac-devel
  libsndfile-devel Vulkan-Headers vulkan-loader $(vopt_if sdl2 SDL2-devel)"
 depends="libspa-alsa>=${version}_${revision} libspa-audioconvert>=${version}_${revision}
  libspa-audiomixer>=${version}_${revision} libspa-control>=${version}_${revision}"
 short_desc="Server and user space API to deal with multimedia pipelines"
-maintainer="Kridsada Thanabulpong <sirn@ogsite.net>"
+maintainer="Stefano Ragni <stefano.ragni@outlook.com>"
 license="MIT"
 homepage="https://pipewire.org/"
 changelog="https://gitlab.freedesktop.org/pipewire/pipewire/-/raw/master/NEWS"
 distfiles="https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/${version}/pipewire-${version}.tar.gz"
-checksum=1d9271e121a5049aef379e9bb7c50524faa6f971e668806637d7b9df1b7cab88
+checksum=2fec0eb47dbfcad51fb8bb3d59c26fe57e09204d63c9d2776070dbdc08aaaaae
+make_dirs="/var/lib/pipewire 0755 pipewire pipewire"
+system_accounts="pipewire"
```

Comment:
Why is this still going with `pipewire`? Packages should ideally stick to the `_*` namespace.

Reply:
upstream uses the `pipewire` user name, but I have no problem adding `_` to be consistent with void's policy

Reply:
`_seatd` wasn't an upstream recommendation either... I think using `_pipewire` is more correct.

Reply:
sorry, fixed